Gemfile

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-ruby '~> 3.1.2'
+ruby '~> 3.2.2'
```

Comment:
I think some gems will break. For example [rspec_api_documentation](https://github.com/zipmark/rspec_api_documentation/pull/507)

Reply:
we should probably remove that gem as well since it's pretty outdated.

Reply:
we need to find a replacement, not just removing it

Reply:
Will create a new pr removing that and exception_hunter, updating pagy and some other stuff I found.

Reply:
rswag seems like the closest thing to it